packages/ui/src/components/UnitInput/index.tsx

```diff
@@ -319,10 +323,8 @@ export const UnitInput = ({
             className={className}
             data-size={size}
           />
-          {error ? <Icon name="alert" sentiment="danger" /> : null}
-          {success && !error ? (
-            <Icon name="checkbox-circle-outline" sentiment="success" />
-          ) : null}
+          {error ? <AlertCircleIcon sentiment="danger" /> : null}
```

Comment:
In this component too the helper has the wrong color

<img width="1114" alt="Screenshot 2024-09-17 at 14 12 05" src="https://github.com/user-attachments/assets/4b72191e-317d-40d6-af81-69116ed9f70a">